src/collections/_documentation/workflow/notifications/index.md

```diff
@@ -3,30 +3,94 @@ title: Notifications
+Notifications in Sentry can be routed to many supported integrations, but by default are aimed at email. Notifications fall into two categories: Alerts and Workflow.
 
-Alerts
+## Alerts
 
-: Generated based upon a project’s alert rules. By default this is only when a new issue is seen. [Learn more]({%- link _documentation/workflow/notifications/alerts.md -%}).
+Each time you create a new project, or a new issue occurs, Sentry creates a default alert rule. Alerts are generated based upon a project’s alert rules. You can customize alerts to the needs of an individual or team.
 
-Workflow
+Alerts have a set of configuration and subscription options. By default, Sentry subscribes you to everything, but you can change this via account settings. You can also configure various personal notifications from your [personal notifications settings](https://sentry.io/settings/account/notifications) and any specific project's settings in the Project Settings page. For more details, see the [full documentation on Alerts]({%- link _documentation/workflow/notifications/alerts.md -%}).
 
-: All activity related to user actions such as resolution, comments, and automatic regressions. [Learn more]({%- link _documentation/workflow/notifications/workflow.md -%}).
+## Workflow
 
-Each category has its own set of configuration and subscription options. By default you’ll be subscribed to everything, but this can be changed via your [account settings](https://sentry.io/settings/account/notifications/). Read about [notification management]({%- link _documentation/workflow/notifications/notification-management.md -%}) for more information.
+Workflow relates to all activity involving user actions such as resolution, comments, and automatic regressions.
+
+Workflow has a set of configuration and subscription options. By default, Sentry subscribes you to everything, but you can change this via your account settings. You can also configure various personal notifications from your [personal notifications settings](https://sentry.io/settings/account/notifications) and any specific project's settings in the Project Settings page. For more details, see the [full documentation on Workflow]({%- link _documentation/workflow/notifications/workflow.md -%}).
 
 ## Issue States
 
-Sentry provides a few states for each issue, which greatly impact how notifications work:
+Sentry provides a few states for each issue, which significantly impacts how notifications work.
+
+### Unresolved
+
+- The default state when an issue is added to the system.
+
+### Resolved
+
+- An issue is marked as resolved when an individual resolves it by manually changing the state on the issue page UI, submitting a fix, or when the project’s auto-resolve feature is configured. Team members are notified via email. Auto-resolve is located in **Project Settings > General Settings**.
```

Comment:
We could expand upon the "submitting a fix" bit - maybe in parens add (e.g. Fixes SENTRY-317) and/or link [here](https://docs.sentry.io/workflow/releases/?platform=go#after-associating-commits) to clarify it's _only_ when you submit a fix in that way.